app/controllers/todos_controller.rb

```diff
@@ -0,0 +1,38 @@
+class TodosController < ApplicationController
+	def index
+	end
+
+	def list
+
+	end
+
+	def create
+		params[:user_id] = current_user.id
+		run Todo::Create do |op|
+			return redirect_to(list_path)
+		end
+		render :list
+	end
```

Comment:
I'm not saying you should implement this now but this can be made more succinct by doing
```
run Todo::Create, params.merge(current_user: current_user) do
    # you can also leave out the block argument (i.e. `op`) when you're not using it here.
end
```
This way, you'd be simply passing `current_user` to the operation for added context. It will will then be available from inside the operation as params[:current_user].